### ISSUE TITLE
Bench2Drive Evaluation scores

### ISSUE THREAD

**Issue by jays-nd** [RenzKa/simlingo#72]
Hi, 

I just had a small doubt regarding the B2D scores. The Bench2Drive closed-loop evaluation scores of "simlingo-base" and "simlingo" in the simlingo paper (Table 2)(https://arxiv.org/pdf/2503.09594) are based on a model trained on a validation set of Town 13? or the training data includes town 13?
The reason I'm asking this is because the paper mention DS scores around 85-85.9 and in the closed issue (https://github.com/RenzKa/simlingo/issues/43), @RenzKa reproduced the scores around 87.5 (where the training data included town 13).

Thanks for your time.

Best,
Jay

**Comment by RenzKa** [comment]
Hi, 
Sorry I might have not fully understood your question. 
Just to double-Check: What’s the difference of this question to your question in issue #43?

Best,
Katrin

**Comment by jays-nd** [comment]
> Hi, Sorry I might have not fully understood your question. Just to double-Check: What’s the difference of this question to your question in issue [#43](https://github.com/RenzKa/simlingo/issues/43)?
> 
> Best, Katrin

Both the questions are the same one. It's just that I didn't received any response in issue #43 thread, so I created a new one and elaborated the question here.

I understand that B2D is a training benchmark and ideally the training set should include all the towns. And from my experiments, I have observed 1-2% increase in average DS when town13 data is added to the training set (note: my experiments are related to TF++, so based on that, I'm comparing the performances between "--setting all" and "--setting 13_withheld")

I hope this clears the doubt about the question. Let me know if you need any additional info.

Best,
Jay

**Comment by RenzKa** [comment]
Thanks for clarifying. I thought this (https://github.com/RenzKa/simlingo/issues/43#issuecomment-3236168801) would have answered your question, and I totally missed your follow-up. Sorry for that. 

So yes, we do include Town13 in our training, and the models in Table 2 are trained including Town 13.

Hope that answers your question. Let me know if not :)

Best,
Katrin

**Comment by jays-nd** [comment]
Thanks for the answer. I am just curious about the ~1.5% difference between DS scores mentioned in the paper and the one which you reported in https://github.com/RenzKa/simlingo/issues/43#issuecomment-3159938161 (is it just because of the variance in the DS?)
Let me know if you have any thoughts on this or else you can close this issue.

**Comment by RenzKa** [comment]
Yes, this should be because of the variance of training and evaluation in CARLA.
